google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java

```diff
@@ -332,23 +334,41 @@ String getServiceNameFromAppEngineWebXml(XmlFile appengineWebXml) {
 
   @Override
   public void generateAppYaml(FlexibleRuntime runtime, Module module, Path outputFolderPath) {
+    UsageTrackerProvider.getInstance()
+        .trackEvent(GctTracking.APP_ENGINE_FLEX_APP_YAML_CREATE)
```

Comment:
+1. That makes sense.  I'd prefer to keep the tracking calls as high up the action stack as possible so that we don't inadvertently fire off a bunch of events through service calls.